release/edgex-taf.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 name: 'edgex-taf'
-version: '2.2.0'
-releaseName: 'kamakura'
+version: '2.3.0'
+releaseName: 'levski'
 releaseStream: 'main'
 lts: false
 repo: 'https://github.com/edgexfoundry/edgex-taf.git'
-commitId: '0eb33292925693c8c97cb37f4ecfa0da53443efa'
+commitId: '299d7aeddaddd8f6b99b88a4c14d83904ed9c601'
```

Comment:
please update it to `3a2c4be1cfe8792113c530d56c2e3db342b553a7` which uses the levski images instead of `main` (dev) images
https://github.com/edgexfoundry/edgex-taf/commits/levski